src/dotnet/Chat.UI.Blazor/Components/LeftPanel/ThinLeftPanel.razor

```diff
@@ -25,6 +26,6 @@
 @code {
     [Inject] private ModalUI ModalUI { get; init; } = null!;
 
-    private void OnCreateButtonClick()
-        => ModalUI.Show(new CreationModal.Model());
+    private void OnCreateNewChat()
```

Comment:
Why don't we use Task as a return type of event handler since ModalUI.Show returns Task and blazor supports Task?

Reply:
Done